src/emulator/CMakeLists.txt

```diff
@@ -49,13 +49,16 @@ endmacro(configure_boost)
 
 configure_boost()
 
+# add_definitions(-DUSE_GDBSTUB)
```

Comment:
Making this an CMake option and put it in CMake config file (```config_file``` ?) would be cool. I would avoid doing this to be honest.